pkg/datamodel/datamodel.go

```diff
@@ -401,6 +401,31 @@ type pbDefinition interface {
 	GetVersion() string
 }
 
+// FeatureScores holds the feature score of each component definition. If a
```

Comment:
This makes sense. The score should be beyond the scope of the component definition. I think this is a good approach until we have the actual ranking data.